week-3/reading-list/script.js

```diff
@@ -21,3 +21,38 @@ const books = [
   },
 ];
 
+function renderBooks(book) {
+  const bookElement = document.createElement("li");
+
+  bookElement.style.backgroundColor = book.alreadyRead ? "green" : "red";
+
+  const titleElement = document.createElement("h3");
+  titleElement.textContent = book.title;
+  bookElement.appendChild(titleElement);
+
+  const authorElement = document.createElement("p");
+  authorElement.textContent = book.author;
+  bookElement.appendChild(authorElement);
+
+  const imageElement = document.createElement("img");
+  imageElement.src = book.bookCoverImage;
+  bookElement.appendChild(imageElement);
+
+  return bookElement;
+}
+
+function readingList() {
```

Comment:
One more naming recommendation. Functions should describe action. This allow us to distinguish functions and variables just even by names. I would suggest rename it to `buildReadingList` or `renderReadingList`.